stdlib/public/core/Sort.swift.gyb

```diff
@@ -86,7 +90,8 @@ func _sort3<C>(
   C : MutableCollection & RandomAccessCollection
   ${"" if p else ", C.Iterator.Element : Comparable"}
 {
-  // Possible starting states for elements at a, b, and c:
+  // Possible starting orderings for elements at `elements[a]`, `elements[b]`,
+  // and `elements[c]`:
```

Comment:
Ah, I completely get it—I don't know why I made that list look so much like a collection! I've updated those comments; please let me know if those make more sense. I'm trying to make the flow of the different cases clear, since there's some subtlety due to keeping the sort stable.